the-incredible-bulk/src/main/java/gov/va/api/health/bulkfhir/service/controller/patient/BulkPatientController.java

```diff
@@ -0,0 +1,192 @@
+package gov.va.api.health.bulkfhir.service.controller.patient;
+
+import gov.va.api.health.bulkfhir.api.internal.BuildStatus;
+import gov.va.api.health.bulkfhir.api.internal.PublicationStatus;
+import gov.va.api.health.bulkfhir.service.controller.publication.DefaultPublicationStatusTransformer;
+import gov.va.api.health.bulkfhir.service.controller.publication.PublicationStatusTransformer;
+import gov.va.api.health.bulkfhir.service.status.StatusEntity;
+import gov.va.api.health.bulkfhir.service.status.StatusRepository;
+import gov.va.api.health.dstu2.api.elements.Narrative;
+import gov.va.api.health.dstu2.api.resources.OperationOutcome;
+import gov.va.api.health.ids.api.IdentityService;
+import gov.va.api.health.ids.api.Registration;
+import gov.va.api.health.ids.api.ResourceIdentity;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import javax.servlet.http.HttpServletRequest;
+import lombok.Builder;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@Validated
+@RestController
+@RequestMapping(
+  value = {"Patient"},
+  produces = {"application/json"}
+)
+class BulkPatientController {
+
+  /** The list of valid output formats for an $export request. */
+  private static List<String> VALID_OUTPUT_FORMATS =
+      List.of("application/fhir+ndjson", "application/ndjson", "ndjson");
+
+  /** The list of valid values for the Accept header for an $export request. */
+  private static List<String> VALID_ACCEPT_HEADER_VALUES =
+      List.of("application/fhir+json", "application/json");
+
+  private final StatusRepository repository;
+
+  private final PublicationStatusTransformer transformer;
+
+  private final String bulkStatusUrl;
+
+  private final IdentityService identityService;
+
+  @Builder
+  BulkPatientController(
+      @Value("${incrediblebulk.url}") String baseUrl,
+      @Value("${incrediblebulk.bulk-status-path:/services/fhir/v0/dstu2/bulk}")
```

Comment:
This path doesn't seem right. The _/services_  part of our paths' is only external, i.e. through adhoc gateway. It is stripped off way before it reaches the applications.